command/daemon.go

```diff
@@ -157,8 +157,11 @@ func daemonCommand(cctx *cli.Context) error {
 		if err != nil {
 			return err
 		}
-		finderOpt := httpfinderserver.WithHomepage(cfg.Addresses.FinderWebpage)
-		finderSvr, err = httpfinderserver.New(finderAddr.String(), indexerCore, reg, finderOpt)
+		finderOpts := []httpfinderserver.ServerOption{
+			httpfinderserver.WithHomepage(cfg.Addresses.FinderWebpage),
+			httpfinderserver.MaxConnections(cfg.Finder.MaxConnections),
+		}
+		finderSvr, err = httpfinderserver.New(finderAddr.String(), indexerCore, reg, finderOpts...)
```

Comment:
Need to set read and write timeouts.
```suggestion
		finderSvr, err = httpfinderserver.New(finderAddr.String(), indexerCore, reg,
			httpfinderserver.WithHomepage(cfg.Addresses.FinderWebpage),
			httpfinderserver.MaxConnections(cfg.Finder.MaxConnections),
			httpfinderserver.ReadTimeout(time.Duration(cfg.Finder.ApiReadTimeout)),
			httpfinderserver.WriteTimeout(time.Duration(cfg.Finder.ApiWriteTimeout)),
		)
```

Also, I think that `cfg.Addresses.FinderWebpage` should be moved to `cfg.Finder.Webpage`.

Reply:
Thanks @willscott @gammazero for the review. 
Updated with read/write timeouts.

Will look at the option movement.